src/main/java/org/prebid/server/bidder/orbidder/OrbidderBidder.java

```diff
@@ -120,4 +123,34 @@ private BidResponse decodeBodyToBidResponse(BidderCall<BidRequest> httpCall) {
             throw new PreBidException(e.getMessage(), e);
         }
     }
+
+    private BidderBid toBidderBid(Bid bid, String cur, List<BidderError> errors) {
+        final BidType bidType;
+        try {
+            bidType = getBidTypeFromMtype(bid.getMtype());
+        } catch (PreBidException e) {
+            errors.add(BidderError.badServerResponse(e.getMessage()));
+            return null;
+        }
+
+        return BidderBid.of(bid, bidType, cur);
+    }
+
+    private static BidType getBidTypeFromMtype(Integer mType) {
```

Comment:
Lets name it shorter, something like `getBidType`.

Reply:
@And1sS : done, I used your naming